[2025-05-07T23:14:42.411+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: graph_analysis.define_categories manual__2025-05-07T23:14:36.293334+00:00 [queued]>
[2025-05-07T23:14:42.416+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: graph_analysis.define_categories manual__2025-05-07T23:14:36.293334+00:00 [queued]>
[2025-05-07T23:14:42.417+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-05-07T23:14:42.427+0000] {taskinstance.py:1327} INFO - Executing <Task(PostgresOperator): define_categories> on 2025-05-07 23:14:36.293334+00:00
[2025-05-07T23:14:42.431+0000] {standard_task_runner.py:57} INFO - Started process 13042 to run task
[2025-05-07T23:14:42.433+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'graph_analysis', 'define_categories', 'manual__2025-05-07T23:14:36.293334+00:00', '--job-id', '292', '--raw', '--subdir', 'DAGS_FOLDER/graph_analysis.py', '--cfg-path', '/tmp/tmp62u6ew30']
[2025-05-07T23:14:42.433+0000] {standard_task_runner.py:85} INFO - Job 292: Subtask define_categories
[2025-05-07T23:14:42.446+0000] {logging_mixin.py:150} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-07T23:14:42.475+0000] {task_command.py:410} INFO - Running <TaskInstance: graph_analysis.define_categories manual__2025-05-07T23:14:36.293334+00:00 [running]> on host 3530b0b864fd
[2025-05-07T23:14:42.542+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='finbest' AIRFLOW_CTX_DAG_ID='graph_analysis' AIRFLOW_CTX_TASK_ID='define_categories' AIRFLOW_CTX_EXECUTION_DATE='2025-05-07T23:14:36.293334+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-07T23:14:36.293334+00:00'
[2025-05-07T23:14:42.542+0000] {sql.py:265} INFO - Executing: 
    ALTER TABLE raw.masked_transactions ADD COLUMN IF NOT EXISTS category VARCHAR;
    
    UPDATE raw.masked_transactions
    SET category = 
        CASE 
            WHEN transaction_type = 'p2p' THEN 'p2p'
            WHEN merchant IN ('Amazon', 'Ozon') THEN 'retail'
            WHEN merchant = 'Uber' THEN 'travel'
            ELSE 'other'
        END
    WHERE category IS NULL;
    
[2025-05-07T23:14:42.543+0000] {crypto.py:83} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-05-07T23:14:42.543+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-05-07T23:14:42.582+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-05-07T23:14:42.588+0000] {sql.py:374} INFO - Running statement: 
    ALTER TABLE raw.masked_transactions ADD COLUMN IF NOT EXISTS category VARCHAR;
    
    UPDATE raw.masked_transactions
    SET category = 
        CASE 
            WHEN transaction_type = 'p2p' THEN 'p2p'
            WHEN merchant IN ('Amazon', 'Ozon') THEN 'retail'
            WHEN merchant = 'Uber' THEN 'travel'
            ELSE 'other'
        END
    WHERE category IS NULL;
    , parameters: None
[2025-05-07T23:14:42.601+0000] {sql.py:383} INFO - Rows affected: 0
[2025-05-07T23:14:42.611+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=graph_analysis, task_id=define_categories, execution_date=20250507T231436, start_date=20250507T231442, end_date=20250507T231442
[2025-05-07T23:14:42.644+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-05-07T23:14:42.670+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
